.travis.yml

```diff
@@ -1,3 +1,13 @@
 language: node_js
 node_js:
   - "14"
+addons:
+  firefox: latest
```

Comment:
out of curiosity, why ff is latest but chrome is stable?

Reply:
The docs for travis don't have any version of firefox named 'stable' https://docs.travis-ci.com/user/firefox or any version of chrome named 'latest' https://docs.travis-ci.com/user/chrome#selecting-a-chrome-version. I think it just mimics what chrome/firefox name files on their site, rather than having a consistent API.